extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -396,7 +396,7 @@ export class AmpAndroidAppBanner extends AbstractAppBanner {
         .then(response => this.parseManifest_(response))
         .catch(error => {
           this.hide_();
-          rethrowAsync(error);
+          rethrowAsync(dev().assertError(error));
```

Comment:
I guess I still don't understand why the new code is an improvement over the old one: it's certainly not easier to read and doesn't really give us more information.